cpp/modmesh/serialization/SerializableItem.hpp

```diff
@@ -89,31 +174,160 @@ std::string to_json_string(const T & value)
     }
 }
 
-}; /* end namespace detail */
+template <typename T>
+void JsonHelper::from_json_string(const std::unique_ptr<JsonNode> & node, T & value)
```

Comment:
Need code comment for the plan to mitigate the output arguments.

Reply:
(I hit review changes button too quickly.)  We can discuss here or discord for how to author the plan.  It may be a simple one.